lib/core/viewmodels/quick_links_viewmodel.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+import 'package:notredame/core/constants/quick_links.dart';
+import 'package:notredame/core/models/quick_link_model.dart';
+import 'package:notredame/ui/views/security_view.dart';
+import 'package:stacked/stacked.dart';
+
+class QuickLinksViewModel extends BaseViewModel {
+  final int numberOfLinks = quickLinks.length;
+
+  List<QuickLink> quickLinkList = quickLinks;
+
+  void onLinkClicked(BuildContext context, QuickLink links) {
```

Comment:
Add some documentation for the function, please

Reply:
done